src/style.css

```diff
@@ -0,0 +1,35 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(-n + 5) {
+  background-image: url("./images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--0 :nth-last-child(-n + 5),
+.stars--1 :nth-last-child(-n + 4),
+.stars--2 :nth-last-child(-n + 3),
+.stars--3 :nth-last-child(-n + 2),
+.stars--4 :nth-last-child(1) {
+  background-image: url("./images/star.svg");
```

Comment:
Everything is fine except one moment. You need to use `:nth-child` only for active stars and you did it above. And that's it.
And for basic stars you only need to move your properties:
```
background-image: url("./images/star.svg");
background-repeat: no-repeat;
background-position: center;
```
to `stars_star` class above and remove all of your `.stars--n :nth-last-child` because they're no longer needed